### ISSUE TITLE
Stage does not have keyboard focus after clicking project preview

### ISSUE THREAD

**Issue by KaidiKelsey** [wordplaydev/wordplay#208]
## Expected behavior
When I type any letter to guess, it should work to let me know how many letters remaining.

## Actual behavior

Provide detailed steps that consistently reproduce the behavior, including any example code. If you can, make the steps as minimal as possible, finding the smallest possible set of steps that still produce the unexpected behavior.

1. Go to 'Galleries'
2. Click on the third game under 'Games'
3. The game does not work

## Screenshots

<img width="1323" alt="Screen Shot 2023-10-17 at 6 18 18 PM" src="https://github.com/wordplaydev/wordplay/assets/97130417/f015233c-47b7-4009-9f89-bb5fbd4535f9">

It asks me to type letters to guess but it does not show any changes. 

## Environment

What platform, OS, browser, and version are you using? Please be exact with version numbers, as different versions can result in a defect happening or not.

Desktop

-   Browser: FireFox
-   Device: macbook

**Comment by amyjko** [comment]
Can you add your Firefox version, operating system, and version? We need that to reproduce the problem.

**Comment by amyjko** [comment]
I can't reproduce. Can you share exactly what letters you're typing and if there is any console log output in the web inspector. (If the site crashes, it can become unresponsive).

**Comment by KaidiKelsey** [comment]
I opened it today, and it becomes this page(below) again so that I cannot go to the game. And for previous error, it just cannot be responsive when typing any letters. 

<img width="1321" alt="Screen Shot 2023-10-18 at 8 34 47 AM" src="https://github.com/wordplaydev/wordplay/assets/97130417/2849b5ab-ac33-4278-ac7f-bd10ebe7bee6">

The version of Firefox is 118.0.2 (64-bit). I open the website through run `npm run dev` in the terminal.

**Comment by amyjko** [comment]
Are you on the latest `main` branch? (Have you done a `git pull` recently?) I ask because the issue with projects opening with all windows collapsed issue was fixed more than a week ago.

You also forgot to provide the console log information that I requested above. Open your browser inspector, go to the console, and report any errors listed in red.

**Comment by KaidiKelsey** [comment]
Yes, I have done git pull recently. 
I cannot reproduce that game page now and don't know how to figure out the issue below:

<img width="1316" alt="Screen Shot 2023-10-18 at 8 52 58 AM" src="https://github.com/wordplaydev/wordplay/assets/97130417/e38c61b1-2d7a-4ea7-b394-a6c4a99788c2">

**Comment by amyjko** [comment]
A few notes:

* According to the URL, you're not viewing the WhatWord game, you're viewing "Adventure". Go back to the gallery and choose the WhatWord game.
* Press "Escape" to leave full screen (or press the full screen button in the top right), and you'll see the project toolbar, where you can show the code and stage windows.
* I'm not certain you have the latest code, since Firefox is rendering the incorrect emoji. That issue was resolved last week. Use `git log` to see recent commits in your branch; you should see commits from me this morning and last night.

**Comment by KaidiKelsey** [comment]
Sorry for viewing the wrong game. I think I figured out the reason why typing letters does not work for me. When I click the game on the gallery page and directly type the letters, it is not responsive. However, it is responsive when I click the game on the gallery page, click the screen, and then type the letters to play it. Is this an issue we have to fix? Because I don't know whether users will do clicking the screen before typing the letters. Thank you for your patience. And sorry for the mistake.
